src/game.h

```diff
@@ -126,6 +130,8 @@ class game
         // May be a bit hacky, but it's probably better than the header spaghetti
         std::unique_ptr<map> map_ptr;
         std::unique_ptr<player> u_ptr;
+        std::unique_ptr<live_view> liveview_ptr;
+        live_view& liveview;
```

Comment:
Is this reference really needed? A quick grep showed only 6 lines which use the reference.